package.json

```diff
@@ -1,6 +1,6 @@
 {
   "scripts": {
-    "format": "prettier --config .prettierrc.json --write !.github ."
+    "format": "npx biome check --write ./web biome.json package.json README.md"
```

Comment:
It's currently just formatting. There are about 35 errors that need to be addressed before we can put the linter in place.